server/src/internalClusterTest/java/org/elasticsearch/action/admin/indices/create/ShrinkIndexIT.java

```diff
@@ -396,7 +397,10 @@ public void testCreateShrinkIndexFails() throws Exception {
 
         refreshClusterInfo();
         // kick off a retry and wait until it's done!
-        ClusterRerouteResponse clusterRerouteResponse = clusterAdmin().prepareReroute().setRetryFailed(true).get();
+        final var clusterRerouteResponse = client().execute(
+            TransportClusterRerouteAction.TYPE,
+            new ClusterRerouteRequest().setRetryFailed(true)
+        ).actionGet(SAFE_AWAIT_TIMEOUT);
```

Comment:
Same question as elsewhere: why do we need `SAFE_AWAIT_TIMEOUT` in this case?

Reply:
Addressed in 21a861522cd97ad49229f628eb2eadd034f63f31